_topics/en/transaction-pinning.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Transaction pinning
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Security Problems
+  - Fee Management
+  - Contract Protocols
+  - Transaction Relay Policy
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Transaction pinning** is a method for making fee bumping
+  prohibitively expensive by abusing node protections against
+  attacks that can waste bandwidth, CPU, and memory.  This can make fee
+  management more difficult in multipart contract protocols (such as LN).
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Nodes such as Bitcoin Core that allow transactions to be replaced
+  (RBF) or packaged with higher-fee child transactions (CPFP) place
+  restrictions on those replacements in order to prevent various DoS
+  attacks.  However, when two or more people each have the ability to
+  fee bump a transaction, this makes it possible for one of them to
+  pin their version of a transaction at one of the limits and prevent
+  other participants from using fee bumping.
+
+  Some of the limits that can be abused to enable transaction pinning
+  include:
+
+  - **[BIP125][] RBF rule #3** requires a replacement transaction
+    pay a higher absolute fee (not just feerate) than the transaction
+    being replaced and any of its children.  This can allow an attacker
+    to attach a large and low-feerate transaction to the transaction
+    they want to pin, forcing any fee bump to pay for pay for the
+    replacement of the large child transaction.  E.g., with the 2019
+    Bitcoin Core defaults, an attacker can require an honest participant
+    pay a minimum of 0.002 BTC to fee bump a transaction (or even
+    greater amounts in some cases).  <!-- 0.002 BTC = 100,000 vbyte tx
```

Comment:
Ha. I was about to leave a review comment here saying 'I haven't checked this number - perhaps leave an HTML comment with your working', but you already have! :stars: